### PR TITLE
Design/edit profile

### LINKED ISSUE #694: Apply design edit page lifebank user
Dep: #566

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #694 

### What does this PR do?
New edit profile lifebank design

### How should this be tested?
- Make run
- edit lifebank profile

image
image
image

### CODE REVIEW

File: webapp/src/language/es.json [hunk 1]
```diff
@@ -259,7 +259,15 @@
...
-      "noUniqueUrl": "Esta direccion esta asociada a otra cuenta"
+      "noUniqueUrl": "Esta direccion esta asociada a otra cuenta",
+      "editTitleLifebank": "Perfil de la cuenta de Lifebank",
+      "information": "La información que se encuentra en este perfil se puede ver en su página de Lifebank. Asegúrese de incluir información detallada para que los donantes puedan encontrarlo fácilmente y contactarte para proceder con la donación...",
+      "officialName": "Nombre Oficial",
+      "contactInformation": "Información de Contacto",
+      "addressInformation": "Datos de Domicilio",
+      "addressDescription": "Describe tu dirección de Lifebank y usa el mapa para seleccionar la ubicación correcta para que los donantes puedan llegar a ti fácilmente.",
+      "imagery": "Imágenes",
+      "imageryDescription": "Escriba un enlace a sus imágenes para que su página de Lifebank se muestre más atractiva para sus donantes. Puede subir imágenes al logotipo de IMGUR.com o la imagen de avatar debe ser una imagen cuadrada, no mayor de 500px o más pesada de 250kb. La imagen del banner debe ser una imagen horizontal, no mayor a 1200px ni más pesada que 1mb."
```
Comment (leisterfrancisco) [inline webapp/src/language/es.json:270]: Im not sure but, isn't missing here the next section:

`"city": "City", "stateProvince": "State/Province", "country": "Country"`

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (leisterfrancisco) [inline yarn.lock:1]: Is this file needed?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Comment by leisterfrancisco** [comment]
@Edgar097 take a look at this.
Also, according to the files changed, it doesn't seems you have changed o modify anything in the frontend, may be some files are missing to push, take a look
image

File: package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```
Comment (JustinCast) [inline package.json:2]: I think you added a dependency when you were at the root level of the project. You need to enter the webapp directory and install it

**Review by JustinCast — CHANGES_REQUESTED** [review]

File: webapp/src/language/es.json (hunk 1 above)
Comment (Edgar097) [reply webapp/src/language/es.json:270]: what do you mean? @leisterfrancisco

File: package.json (hunk 3 above)
Comment (Edgar097) [reply package.json]: Is that the correct way? @JustinCast

File: webapp/src/language/es.json (hunk 1 above)
Comment (leisterfrancisco) [reply webapp/src/language/es.json:270]: @Edgar097 so, in English version exist `"city": "City", "stateProvince": "State/Province", "country": "Country"` inside of the editProfile tag, but in Spanish it doesn't

Comment (leisterfrancisco) [reply webapp/src/language/es.json:270]: Take a look:
image

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by steph222 — APPROVED** [review]

**Review by leisterfrancisco — APPROVED** [review]
